app/views/content/tips-on-applying-for-teacher-training.md

```diff
@@ -46,11 +46,11 @@ interviews. A [teacher training adviser](/tta-service) can help you with all of
 
 ## Browse courses
 
-You can [search for most postgraduate teacher training courses on GOV.UK](https://www.gov.uk/find-postgraduate-teacher-training-courses).
+Take a look through most [postgraduate teacher training courses](https://www.gov.uk/find-postgraduate-teacher-training-courses).
 
 ## Where to apply
 
-You can use [Apply for teacher training](https://www.gov.uk/apply-for-teacher-training) to apply for postgraduate teacher training.
+[Apply for teacher training](https://www.gov.uk/apply-for-teacher-training).
```

Comment:
Will this hang a bit on the page. Do we need to put more of a sentence around it? Or possibly change the title?

Reply:
maybe merge this section with 'what to expect: the application form'?

The first sentence in that section could then just have an apply link, rather than having this short section hanging on?
e.g 'When you **apply for teacher training**, you'll need to give details about:'

Or do you think we need a distinct section for applying?

Reply:
@gemmadallmandfe thoughts on this? :)

Reply:
I agree this hangs a bit. I'd suggest keeping a separate section for applying but put a bit of wording around it - I've made a suggestion.

I'd suggest just getting this update in for now but I think we will revisit this page more fully as we look to integrate GIT better with Find and Apply in the New Year.